pkg/mark/renderer/blockquote.go

```diff
@@ -81,6 +85,19 @@ func ParseBlockQuoteType(node ast.Node, source []byte) BlockQuoteType {
 			if node.Kind() == ast.KindText {
 				n := node.(*ast.Text)
 				t = ClassifyingBlockQuote(string(n.Text(source)))
+				// If the node is a text node but classification returned none do not give up!
+				// Find the next sibling node, if it is also a text node and the original node text value is '['
+				// It means with high degree of confidence that the original md doc contains a Github alert type of blockquote
+				// Classifying the next text type node will confirm that.
+				if t == None {
+					nextSiblingNode := node.NextSibling()
+					if nextSiblingNode.Kind() == ast.KindText {
+						nx := nextSiblingNode.(*ast.Text)
+						if string(n.Text(source)) == "[" {
```

Comment:
We should also check for the exclamation mark after it and the closing "]" here, otherwise this will match on

```
[!NOTE
```

and 
```
[NOTE]
```

Reply:
Rethinking this a bit, could we tweak the regular expressions instead of parsing this as a single character?

Reply:
I was using a mix of both